source/pervasive/arithmetic/div_mod.rs

```diff
@@ -0,0 +1,2397 @@
+//! This file contains proofs related to integer division (`/`) and
+//! remainder aka mod (`%`). These are part of the math standard library.
+//!
+//! It's based on the following file from the Dafny math standard
+//! library:
+//! `Source/DafnyStandardLibraries/src/Std/Arithmetic/DivMod.dfy`.
+//! That file has the following copyright notice:
+//! /*******************************************************************************
+//! * Original: Copyright (c) Microsoft Corporation *
+//! SPDX-License-Identifier: MIT * * Modifications and Extensions:
+//! Copyright by the contributors to the Dafny Project *
+//! SPDX-License-Identifier: MIT
+//! *******************************************************************************/
+use crate::calc_macro::*;
+#[allow(unused_imports)]
+use builtin::*;
+use builtin_macros::*;
+
+verus! {
+
+#[allow(unused_imports)]
+#[cfg(verus_keep_ghost)]
+use crate::arithmetic::internals::div_internals::{
+    div_recursive,
+    lemma_div_induction_auto,
+    div_auto,
+    div_pos,
+    lemma_div_auto,
+};
+use crate::arithmetic::internals::div_internals_nonlinear as DivINL;
+#[cfg(verus_keep_ghost)]
+use crate::arithmetic::internals::mod_internals::{
+    lemma_div_add_denominator,
+    lemma_mod_auto,
+    mod_recursive,
+};
+use crate::arithmetic::internals::mod_internals_nonlinear as ModINL;
+#[cfg(verus_keep_ghost)]
+use crate::arithmetic::internals::mul_internals::{
+    lemma_mul_auto,
+    lemma_mul_induction,
+    lemma_mul_induction_auto,
+};
+#[cfg(verus_keep_ghost)]
+use crate::arithmetic::internals::general_internals::{is_le};
+#[cfg(verus_keep_ghost)]
+use crate::arithmetic::math::{add as add1, sub as sub1, div as div1};
+use crate::arithmetic::mul::*;
+
+/*****************************************************************************
+   * Division:
+   *****************************************************************************/
```

Comment:
Maybe it's just how GitHub is showing it, but the 2nd and 3rd lines of this comment (and the one further below) seem to be oddly indented.  FWIW, I'd also be inclined to cut the colon.